RELEASING.md

```diff
@@ -1,3 +1,69 @@
 ## Releasing
 
-This project follows [OpenSearch project branching, labelling, and releasing](https://github.com/opensearch-project/.github/blob/main/RELEASING.md).
\ No newline at end of file
+This project follows [OpenSearch project branching, labelling, and releasing](https://github.com/opensearch-project/.github/blob/main/RELEASING.md).
+
+## Runbook
+
+### Overview
+
+The OpenSearch project releases as versioned distributions of OpenSearch, OpenSearch Dashboards, and the OpenSearch plugins. This runbook describes in detail how to perform an end-to-end major/minor/patch version release for OpenSearch Dashboards project as the release manager (RM) of this release version. RM is responsible for updating the release status broadly on the release tracking issue on github.
+
+#### Important Dates
+
+- Next minor release: [Version 2.4.0 (November 10, 2022)](https://github.com/orgs/opensearch-project/projects/1#column-19044869)
```

Comment:
NIT: I would remove these dates as this is maintained in GitHub issues and by the build repo and subject to be delayed if any security issues occur.

Reply:
You can just refer to https://opensearch.org/releases.html